### DISCUSSION TITLE
[TwigComponent] Error in debug:twig-component ?

### DISCUSSION THREAD

**Discussion by clussiana** [symfony/ux#2324] [Questions & Answers]
Hello, I saw that it was fixed in #2088 but under Windows environment it still breaks, I thinks it's because we are replacing only `/` but not `\` when rewriting components paths.

**Comment by smnandre** [comment]
Hello @clussiana !

Could you provide an example with the values of path and where it seems to miss a conversion ? 

Even more, would you like to send a PR to fix this error ?

**Reply by clussiana** [reply]
Here is the example : 
For a component `Card/Card.html.twig`, it is translated into `Card\Card` instead of `Card:Card`
Then it can't find a path to this component because the path is `C:\...\project/templates/bundles/TailwindBundle/components\` instead of `C:\...\project/templates/bundles/TailwindBundle/components` (because the last backslash is not trimmed)

Sure, I'll ldo a PR, do I need to create the associated issue ?

**Reply by smnandre** [reply]
> Sure I'll do a PR

Great, thank you! 

> do I need to create the associated issue ?

Not at all, but if you can include a test that'd be great
(event a mocked/reflection one - we don't run tests on windows currently)
